library/src/main/java/org/xmtp/android/library/frames/FramesErrors.kt

```diff
@@ -0,0 +1,5 @@
+package org.xmtp.android.library.frames
+
+class FramesApiError(message: String, val status: Int) : Exception(message)
```

Comment:
I think you can just delete this whole file. But if you want to keep it then I would have it extend the `XMTPException` instead of exception.